doc/community-build/cloud/AWS/README.md

```diff
@@ -77,19 +77,19 @@ This file consists of the procedure to compile complete CORTX stack and Deploy o
 
 ### 1. CORTX Build
 
-   We will use [cortx-build](https://github.com/Seagate/cortx/pkgs/container/cortx-build) docker image to compile entire CORTX stack. Please follow [CORTX compilation](https://github.com/Seagate/cortx/blob/main/doc/community-build/Generate-Cortx-Build-Stack.md) steps for compilation.
+   We will use [cortx-build](https://github.com/Seagate/cortx/pkgs/container/cortx-build) docker image to compile entire CORTX stack. Please follow [CORTX compilation](https://github.com/Seagate/cortx/blob/main/doc/community-build/archive/Generate-Cortx-Build-Stack.md) steps for compilation.
 
 ### 2. CORTX Deployment
 
-   After CORTX build is ready follow [CORTX Deployment](https://github.com/Seagate/cortx/blob/main/doc/community-build/ProvisionReleaseBuild.md) to deploy CORTX on AWS instance. Please exclude SELINUX and Hostname setup steps.
+   After CORTX build is ready follow [CORTX Deployment](https://github.com/Seagate/cortx/blob/main/doc/community-build/archive/ProvisionReleaseBuild.md) to deploy CORTX on AWS instance. Please exclude SELINUX and Hostname setup steps.
 
 ### 3. OnBoarding 
 
    Configure CORTX by following [Onboarding](https://github.com/Seagate/cortx/blob/main/doc/Preboarding_and_Onboarding.rst) steps.
 
 ### 4. Testing IO operations 
 
-   We will need separate AWS instance to test IO operations. Follow [IO Test using AWS ](https://github.com/Seagate/cortx/blob/main/doc/integrations/AWS_EC2.md#step-7-create-another-ec2-instance-to-access-to-act-as-the-s3-client) and subsequent steps. 
+   We will need separate AWS instance to test IO operations. Follow [IO Test using AWS ](https://github.com/Seagate/cortx/blob/main/doc/integrations/AWS_EC2/README.md) and subsequent steps. 
```

Comment:
@mukul-seagate11, looks like it pointing to cortx setup using ec2 and not IO using AWS. do we have another link to direct test point?

Reply:
@gauravchaudhari02, it the same link for both where IO section is included

Reply:
No, other then this unable to find the IO refs using AWS